### PR TITLE
add:image,input

### PR DESCRIPTION
**Pull request by thegeorgenikhil**
added images and input components
added utility classes for border-radii

### CODE REVIEW

File: components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            </p>
+            <div class="display-container flex justify-center wrap gap-20">
+              <img class="img-fluid rounded-lg" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+              <img class="img-fluid rounded-md" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+              <img class="img-fluid rounded-sm" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+              <img class="img-fluid rounded-xs" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+            </div>
```
Comment (anuj-kumary) [inline components/Image/image.html]: Always give alt (alternate) in (avatar, badge).

File: components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.input-group label {
+  font-size: var(--text-sm);
+  margin-bottom: 4px;
+  font-weight: var(--font-medium);
+}
+
+.input {
+  font-family: "Inter", sans-serif;
```
Comment (anuj-kumary) [inline components/Input/input.css]: Try to use font family as a variable don't hard code

Comment (thegeorgenikhil) [reply components/Input/input.css]: Yup!
The font is already defined in the root so using inherit would be better right?

File: components/Image/image.html (hunk 1 above)
Comment (thegeorgenikhil) [reply components/Image/image.html]: On it!

File: components/Input/input.css (hunk 2 above)
Comment (thegeorgenikhil) [reply components/Input/input.css]: Done ty!

**Comment by Shah-Arsalan** [comment]
I can't view the images component . Can you check once?

**Comment by thegeorgenikhil** [comment]
@Shah-Arsalan 
https://deploy-preview-4--flow-css.netlify.app/components/image/image
url is working for me. Can you specify a little more?

**Review by surajmore98 — APPROVED** [review]
looks all good. documentation site's design is good!

**Comment by thegeorgenikhil** [comment]
@Shah-Arsalan @anuj-kumary  Hey can you guys check again, I have made some changes.

**Comment by Shah-Arsalan** [comment]
yes yes . I can access now . Looks good too

**Review by anuj-kumary — APPROVED** [review]

**Pull request merged**